xbmc/FileItem.cpp

```diff
@@ -4273,3 +4273,14 @@ bool CFileItem::HasVideoExtras() const
   }
   return false;
 }
+
+bool CFileItem::IsVideoAssetNav() const
+{
+  // @todo maybe in the future look for prefix videodb://movies/videoversions in path instead
+  // @todo better encoding of video assets as path, they won't always be tied with movies.
+  const CURL itemUrl{GetPath()};
```

Comment:
Maybe add a check for being a videodb item first? `if (IsVideoDb()) ...` This is cheap and will avoid parsing of the URL etc. in many cases.